.cirrus.yml

```diff
@@ -849,6 +870,15 @@ local_system_test_task: &local_system_test_task
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*' &&
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:BUILD.*' &&
         $CIRRUS_CHANGE_TITLE !=~ '.*CI:MACHINE.*'
+    # skip: - when PR (we never want to skip on nightly tests)
+    #       - when no danger files are changed
+    #       - when no system test code is changed
+    #       - NOT (source code is changed AND not only test files)
+    skip: &skip_system_test >-
+        $CIRRUS_PR != '' &&
+        !changesInclude('.cirrus.yml', 'Makefile', 'contrib/cirrus/**', 'vendor/**', 'hack/**') &&
+        !changesInclude('test/system/**') &&
+        !(changesInclude('**/*.go', '**/*.c') && !changesIncludeOnly('test/**', 'pkg/machine/e2e/**'))
```

Comment:
Similar to my bud comments. Is it possible for changes to something-else.go to break podman-machine tests?

Reply:
Well I cannot say 100% no. Sure there is risk that some day something weird happens or we update the file structure. Nightly runs should catch things and I also count on reviewers to see that machine tests are marked skipped but they changed some machine related code.

Reply:
I think relying more on the nightly cron-runs is probably okay for complex cases like this.